crates/flowctl/src/connector.rs

```diff
@@ -40,22 +40,16 @@ pub fn docker_spawn(image: &str, args: &[&str]) -> anyhow::Result<(Child, TempDi
     let child = Command::new("docker")
         .args([&[
               "run",
-              "--interactive",
-              "--init",
               "--rm",
-              "--log-driver=none",
-              "--mount",
-              &format!("type=bind,source={host_inspect_str},target={target_inspect}"),
-              "--mount",
-              &format!("type=bind,source={host_connector_init_str},target={target_connector_init}"),
+              "--mount", &format!("type=bind,source={host_connector_init_str},target={target_connector_init}"),
+              "--entrypoint", target_connector_init,
+              "--env", "LOG_LEVEL=trace",
 		      "--publish", &format!("0.0.0.0:{}:{}/tcp", port, CONNECTOR_INIT_PORT),
-              "--entrypoint",
-              target_connector_init,
+              "--mount", &format!("type=bind,source={host_inspect_str},target={target_inspect}"),
               image,
               &format!("--image-inspect-json-path={target_inspect}"),
               &format!("--port={CONNECTOR_INIT_PORT}"),
         ], args].concat())
```

Comment:
Updated the arguments here to match the ones in `container.go`

Reply:
rust fmt? perhaps sort --flags ?

Also remove LOG_LEVEL=trace ? that's going to be _very_ verbose.